contracts/src/spaces/facets/membership/metadata/MembershipMetadata.sol

```diff
@@ -12,7 +14,12 @@ import {LibString} from "solady/utils/LibString.sol";
 import {ERC721ABase} from "contracts/src/diamond/facets/token/ERC721A/ERC721ABase.sol";
 import {Facet} from "contracts/src/diamond/facets/Facet.sol";
 
-contract MembershipMetadata is ERC721ABase, Facet {
+contract MembershipMetadata is IMembershipMetadata, ERC721ABase, Facet {
+  /// @inheritdoc IMembershipMetadata
+  function refreshMetadata() external {
+    emit IERC4906.BatchMetadataUpdate(1, type(uint256).max);
```

Comment:
shouldn't this be from 0? here is the zero hnt token https://testnets.opensea.io/assets/base-sepolia/0x9bd136d3155316d5d6dfcd4f87d5ee1a3ea8f22d/0